static/app/components/replays/walker/urlWalker.tsx

```diff
@@ -62,5 +57,5 @@ function urlToCrumb(url: string) {
     color: 'green300',
     timestamp: undefined,
     data: {to: url},
-  } as BreadcrumbTypeNavigation;
+  } as Crumb;
```

Comment:
just as another nit pick

prefer return types to casting if possible

Reply:
NP